application-confluence-migrator-ui/src/main/resources/Confluence/Tools/Filter.xml

```diff
@@ -152,7 +152,7 @@
   'input' : {
     'verbose' : 'false',
     'usersEnabled' : 'false',
-    'prefixedMacros' : 'chart, gallery',
+    'prefixedMacros' : 'chart, gallery, attachments',
```

Comment:
I feel that this is the wrong logic, prefixed should be the default and only very specific macros (which very good reasons) should be unprefixed...